docs/index.md

```diff
@@ -1,125 +1,18 @@
-# X Components Archetype
+# Interface X Archetype
```

Comment:
Should we take the chance and use `Interface X` instead of `X Components` in other places too? I'm referring mainly to the `README`.

Reply:
And the i18n.md